Android/GoogleDagger/source/Dagger/Dagger.csproj

```diff
@@ -12,19 +12,20 @@
     <RootNamespace>Annotations</RootNamespace>
     <AndroidClassParser>class-parse</AndroidClassParser>
     <AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+    <NoWarn>CS0108,CS0109,CS0114,CS0618</NoWarn>
   </PropertyGroup>
 
   <PropertyGroup>
     <PackageId>Xamarin.Google.Dagger</PackageId>
     <Title>Google Dagger for Xamarin.Android</Title>
-    <PackageDescription>Xamarin.Android bindings for Google Dagger</PackageDescription>
+    <Summary>Xamarin.Android bindings for Google Dagger</Summary>
```

Comment:
This needs to be `<PackageDescription>` No idea why things are different in the csproj, but hey. https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packagedescription

Reply:
Added.